screens/New&Hot/NewHot.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Text, View } from 'react-native'
```

Comment:
The name for this file/directory shouldn't be `New&Hot/NewHot.tsx` but should be `NewAndHot`